### ISSUE TITLE
slds/enforce-wcag-rules error text doesn't replace variables correctly

### ISSUE THREAD

**Issue by aheber** [salesforce-ux/stylelint-sds#19]
A CSS file violating slds/enforce-wcag-rules generates a warning. The text label is 
```
Consider replacing "${oldValue}" with a similar color "${newValue}" styling hook to provide better color contrast
```

Meaning `replacePlaceholders` is expecting an `args` object with the properties `oldValue` and `newValue`

In `enforce-wcag-rules/index.ts` the args object has the properties `color` and `closestHook`.

This discrepancy between replacement params and supplied args means the warning text outputs as 

`Consider replacing "" with a similar color "" styling hook to provide better color contrast  slds/enforce-wcag-rules`

Changing
```javascript
const messages = utils.ruleMessages(ruleName, {
  rejected: (color: string, closestHook: string) =>
    replacePlaceholders(errorMsg, { color, closestHook} ),
    // `Replace the "${color}" value with any styling hook mentioned below "${closestHook}" instead.`,
  suggested: (color: string) =>
    `The "${color}" static value has no replacement styling hook.`,
});
```

for
```javascript
const messages = utils.ruleMessages(ruleName, {
  rejected: (oldValue: string, newValue: string) =>
    replacePlaceholders(errorMsg, { oldValue, newValue} ),
    // `Replace the "${oldValue}" value with any styling hook mentioned below "${newValue}" instead.`,
  suggested: (color: string) =>
    `The "${color}" static value has no replacement styling hook.`,
});
```

brings the messages function in line with the other rules and correctly matches the expected args in the error messages.

**Comment by aheber** [comment]
I think was closed by #30
